specification.markdown

```diff
@@ -320,7 +320,7 @@ The path defines the allowed storage locations of the `EBML Element` within an `
 The `path` attribute is REQUIRED.
 
 ```
-EBMLFullPath             = EBMLElementOccurrence "(" EBMLReferencePath ")"
+EBMLFullPath           = EBMLElementOccurrence "(" EBMLReferencePath ")"
```

Comment:
No, I don’t have a proposal having less U. As Dave said, only shorter names would prevent it. That said, this fixes the issue of the «miniature pages».